<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LoginPageLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size:15px;
}</string>
    </property>
    <property name="text">
     <string>Login Page</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoginNameLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter username: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginNameLEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LoginPassLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginPassLEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginPshButtom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Don't have an account? </string>
    </property>
   </widget>
   <widget class="QPushButton" name="SignUpPshButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoginErrorLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Error: wrong username or password</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
